packages/api/test/addbuild-getbuild.js

```diff
@@ -361,6 +366,70 @@ describe('Add-Build', () => {
     });
   });
 
+  describe('GetExportHandler', async () => {
+    it('Can get a csv export', async () => {
+      const resp = await fetch('http://localhost:3000/api/repo/nodejs/node/csv');
+
+      const respText = await resp.text();
+
+      const linesReal = respText.split(/\n/);
```

Comment:
Added tracking issue for this, since I think it would be easier to add after everything is merged since I added another test case using a file in this [pr](https://github.com/GoogleCloudPlatform/flaky-service/pull/125)

https://github.com/GoogleCloudPlatform/flaky-service/issues/128

Reply:
👍 sounds good to me @StuartRucker